src/System.IO.FileSystem.Watcher/tests/InternalBufferOverflowException.cs

```diff
@@ -0,0 +1,39 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using Xunit;
+
+namespace System.IO.Tests
+{
+    public static class InternalBufferOverflowExceptionTests
+    {
+        [Fact]
+        public static void DefaultConstructor()
+        {
+            InternalBufferOverflowException ide = new InternalBufferOverflowException();
+
+            Assert.NotNull(ide.Message);
+        }
+
+        [Fact]
+        public static void MessageConstructor()
+        {
+            string message = "MessageConstructor";
+            InternalBufferOverflowException ide = new InternalBufferOverflowException(message);
+
+            Assert.Equal(message, ide.Message);
+        }
+
+        [Fact]
+        public static void MessageInnerExceptionConstructor()
+        {
+            string message = "MessageConstructor";
+            Exception innerException = new Exception();
+            InternalBufferOverflowException ide = new InternalBufferOverflowException(message, innerException);
+
+            Assert.Equal(message, ide.Message);
+            Assert.Same(innerException, ide.InnerException);
+        }
+    }
```

Comment:
It's the namespace that contains BinaryFormatter.  BinaryFormatter is in the System.Runtime.Serialization.Formatters assembly.